src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,74 @@
+package mate.academy;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import mate.academy.exceptions.DataProcessingException;
+
+public class ConnectionUtil {
+
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/test";
+    private static final Properties DB_PROPERTIES;
+
+    static {
+        DB_PROPERTIES = new Properties();
+        String user = System.getenv("DB_USER");
+        String password = System.getenv("DB_PASSWORD");
+
+        if (user == null || password == null) {
+            throw new DataProcessingException(
+                "Database credentials are not set in environment variables", null);
```

Comment:
The DataProcessingException is thrown when environment variables for database credentials are not set. It might be more appropriate to use a more specific exception or handle this case differently, such as providing a default value or logging a warning.